server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+
+const { startDatabase, stopDatabase, isConnected } = require('./database');
+const app = express();
```

Comment:
It would be beneficial to add error handling middleware to the Express app. This can help catch and handle any errors that might occur during request processing. [medium]